README.md

```diff
@@ -1,28 +1,108 @@
+   - See https://developer.android.com/ndk/guides/sdk-versions for more information.
+
+
+**versionCode**
+   - A positive integer that serves as the internal version number of the app.
+   - Used to differentiate between newer and older versions, with higher numbers indicating more recent versions.
+   - Not visible to users, as the versionName attribute is used for display.
+   - Prevents downgrading by blocking the installation of APKs with lower versionCodes than the one installed.
+   - Important to increment for each app update.
+   - Maximum allowable value on Google Play: 2100000000.
+   - Reuse of versionCode for Play Store uploads is not permitted.
+   - See https://developer.android.com/studio/publish/versioning for more information.
+
+
+**versionName**
+   - The version number shown to users, specified as a string.
+   - Flexible format, commonly used as <major>.<minor>.<point> or other version identifiers.
+   - The primary version identifier for end users.
+   - See https://developer.android.com/studio/publish/versioning for more information.
+
+
+**package**
+   - Represents the app's unique application ID, formatted as a Java package name.
+   - Can include uppercase and lowercase letters, numbers, and underscores, but must start with a letter.
+   - Modifying this value essentially creates a new application, impacting updates and data transfer.
+   - In AGP 7.3+, avoid setting this directly in the source manifest.
+   - See https://developer.android.com/guide/topics/manifest/manifest-element for more information.
+
+
+## Coming soon
+
+**compileSdkVersion**
+   - This property doesn't impact NDK builds as API availability for NDK is determined by minSdkVersion.
+   - In NDK, C++ symbols are resolved at library load time, unlike the lazy resolution in Java.
+   - Utilizing symbols not present in the minSdkVersion can lead to library loading failures on OS versions lacking the newer API.
+   - Recommended approach for new apps is to use the latest available version. For existing apps, update to the latest version as per development needs.
+   - See https://developer.android.com/ndk/guides/sdk-versions for more information.
+
+
+**compileSdkVersionCodename**
+   - Reflects the development codename of the Android framework used for compiling the app.
+   - Compile-time equivalent of Build.VERSION.CODENAME.
+   - See https://developer.android.com/reference/android/content/pm/ApplicationInfo#compileSdkVersionCodename for more information.
+
+
+**platformBuildVersionCode**
+   - Description unavailable.
+
+
+**platformBuildVersionName**
+   - Description unavailable.
+
 
 ## Usage
 
+```bash
+# Use the following order for entering attribute names, values, and the file name:
+   $ ./androidmanifest-changer [attribute-flag] [value] [file_name]
+   Example: $ ./androidmanifest-changer -minSdkVersion 30 app.aab
+   Flags should precede the file name, and multiple flags can be used in a single command.
```

Comment:
Since this is not valid bash anymore, could you please split it up into separate bash sections with normal text in between?

Also, I think showing a simple example like in the last two cases is easier to understand than this first section. One can already see the pattern while the `[attribute-flag] [value]` looks like it would only support setting a single attribute.

Reply:
Oh and since the section about supported attributes has expanded so much, it could be better to move the Usage section to the top. Otherwise people are greeted with lots of details before even having understood what this tool does.

Reply:
Thank you for the feedback, @wkornewald. I agree with your suggestions. I am currently making the changes to the documentation. This includes splitting the bash script into separate sections with explanatory text in between, adding simpler examples for better understanding, and moving the Usage section to the top for immediate clarity. Your input is greatly appreciated!

Reply:
Probably your changes for the README also got lost?

Reply:
I apologize. I made some mistakes while attempting to revert some of the changes. I will try to get it sorted tonight.